### PR TITLE
Polyhedron demo: some fixes

### LINKED ISSUE #3420: some CGAL demo issues
## Issue Details

- [x] Recenter (ctrl R) and trying to move the item immediately after (roughly within a 1s window) with right click is glitchy and forces the item to be centered again.
- [x] `Set different colors` paints it black if a `*.cgal.polylines.txt` file is present.
- [x] The detection of which simplex is eligible in Euler operation seems broken. Take for example [this](https://gist.github.com/MaelRL/0efdfda1246febbc147a70488e88efb2) and try to split all the edges multiple times. Very quickly some edges do not allow splitting anymore. There are also some strange messages about degenerate faces showing up during Euler processes that shouldn't create any (like edge splitting).

- [x] It would be nice if multiple point clouds could get their sizes changed at the same time.

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes
This PR fixes some bugs in colors and selection, and adds Points Size and Normals Lentght to the list of actions that can be changed simultaneously on several items.
## Release Management
* Based on #3225
* Issue(s) solved (if any): fix #3420 , partially fix#3430

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Scene.cpp [hunk 1]
```diff
@@ -1074,64 +1074,174 @@ bool Scene::dropMimeData(const QMimeData * /*data*/,
...
+  for(int i=1; i<sorted_lists.size(); ++i)
+  {
+    QList<int>& list = sorted_lists[i];
+    if(list.isEmpty())
+      continue;
+    Scene_group_item* group = qobject_cast<Scene_group_item*>(this->item(i));
+    if(!group)
+      continue;
+    std::sort(list.begin(), list.end(),
+              [this, group](int a, int b) {
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Scene.cpp]: This triggers the following warning:
`warning: lambda capture 'this' is not used [-Wunused-lambda-capture]`

File: Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Mesh_segmentation_plugin.cpp [hunk 2]
```diff
@@ -377,7 +377,7 @@ void Polyhedron_demo_mesh_segmentation_plugin::colorize_sdf(
         }
         put(pidmap, *facet_it, static_cast<int>(patch_id));
     }
-    item->setItemIsMulticolor(true);
+    item->setItemIsMulticolor(true, false);
```
[the remark this answers is not in the document]
Comment (maxGimeno) [reply Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Mesh_segmentation_plugin.cpp]: It's the most convenient way I found. I explained it in a commentary in the surface_mesh_item header. If you have a better idea I'm all hears.

[the remark this answers is not in the document]
Comment (lrineau) [reply Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Mesh_segmentation_plugin.cpp]: The *most convenient way*...

The right way is obvious, in my opinion: you have two properties:
  - multicolor,
  - compute the colors automatically.

So... there must be two sets of getters and setters:
  - `itemIsMulticolor()`
  - `itemComputeColorsAutomatically()` (or something like that).

That way the API is easy to read at the point where it is called.

**Comment by sloriot** [comment]
@lrineau are you going to review this PR? (I just saw you added the _not yet approved_ flag). The CGAL-4.14-Ic-43 testsuite looked good.

**Comment by lrineau** [comment]
The changes of the PR look fine, after https://github.com/CGAL/cgal/pull/3424/commits/9dc71d3a61bca184b1324b57b8dbe2db85fe7b8. You can merge it.

**Pull request merged**